api/lib/opentelemetry/trace/span_id.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Trace
+    # A class that represents a trace identifier. A valid span identifier is an
+    # 8-byte array with at least one non-zero byte.
```

Comment:
This is technically incorrect. The identifier here is a integer, not an 8-byte array.